uat/testing-features/src/main/java/com/aws/greengrass/steps/FileSteps.java

```diff
@@ -104,6 +105,13 @@ public void arrangeComponentLogFiles(int numFiles, String componentName) throws
         scenarioContext.put(componentName + ACTIVEFILE, logsDirectory.resolve(fileName).toAbsolutePath().toString());
     }
 
+    @Given("A log directory for component {word}")
```

Comment:
Like the way how you specify the log directory.
A little suggestion: May change to something like "Create a log directory for component {word}".

Reply:
We don't actually create anything here. This is more for registering the path in the context